src/app/cli/src/tests/coda_five_nodes_test.ml

```diff
@@ -4,21 +4,36 @@ open Coda_base
 
 let name = "coda-five-nodes-test"
 
+let runtime_config =
+  lazy
+    ( (* TODO: Decide on a profile for this test.
+         (It has never been used on CI.)
+      *)
+      "{}" |> Yojson.Safe.from_string |> Runtime_config.of_yojson
```

Comment:
Let's delete this file if it's unused

Reply:
This is from your PR #1792; will drop it if you're happy that it's okay.